### ISSUE TITLE
PDF.js v1.1.366 (build: 9e9df56) Bericht: Invalid PDF structure

### ISSUE THREAD

**Issue by achibula** [mozilla/pdf.js#7091]
Mozilla Firefox browser doesn't show any preview anymore from PDF documents in Doccle. It says always invalid PDF structure. This is not possible because i can open them with ALL PDF programs!!!
I am a Linux Mint 17.3 Cinnamon 64bit user!

**Comment by Snuffleupagus** [comment]
*Cross-posted in https://bugzilla.mozilla.org/show_bug.cgi?id=1257832?*

When reporting an issue, please make sure that you read https://github.com/mozilla/pdf.js/blob/master/CONTRIBUTING.md. Unfortunately this issue is missing the necessary steps required to reproduce it!
E.g. what is "Doccle"? A *link* to a test-case would help here.

However, looking at the screenshot attached in Bugzilla, it looks to me like that web page is using a custom version of PDF.js, and *not* the one included in Firefox.
The `Version` field in the bug says `45 Branch`, and the version of PDF.js that is included in that particular Firefox release is actually `1.3.76` (i.e. much newer than `1.1.366`).
Furthermore, the toolbar seen in the screenshot isn't the standard one, and errors aren't reported in that way in the PDF.js version included in Firefox.

**In summary:** Based on the information provided so far, this looks like an issue with a *custom* PDF.js deployment, and not with the PDF.js library (or the PDF.js version included in Firefox).

**Comment by achibula** [comment]
I believe your reply but explain me the following:
When I visit the same website www.doccle.be in Opera or Chromium browser it doesn't give any error.
Is it that both browsers don't use pdf.js system? Or something else?

**Comment by Snuffleupagus** [comment]
> When I visit the same website www.doccle.be in Opera or Chromium browser it doesn't give any error.

Since you did not provided a link to the specific website where this happens, we won't be able to provide any help or insight here!

> Is it that both browsers don't use pdf.js system? Or something else?

Given that this looks to be an issue with a *custom* PDF.js deployment, we're not going to be able to provide a meaningful answer here (again, since no link to a non-working example was provided).

Based on the information you've provided so far, we're unfortunately not able to help any further.
I'd suggest that you contact the website directly, since this sounds like an issue on their end.

Closing as incomplete for now; please provide a link to *the exact* web page where this happens in order to reopen!

---
For future reference: It's enough to open an issue in *either* GitHub *or* Bugzilla.

**Comment by PLQin** [comment]
```
const { success, data } = await getPDF({ fileId:1 });
if (success) {
    const blob = new Blob([data], {
        type: `application/pdf;charset-UTF-8`, // word文档为msword,pdf文档为pdf
    });
    const objectURL = URL.createObjectURL(blob);
    this.pdfUrl = objectURL;
}
```

and set request `responseType: 'blob' `
